### PR TITLE
New UI design implementation

### LINKED ISSUE #4111: Implement new UI format for top of tag pages
This is part of a set of issues to take the design work produced by Kelvin at https://publiclab.org/notes/kelv/11-21-2018/tag-page, break it into design "pieces", and implement a number of them, while preserving our default Bootstrap styles: https://getbootstrap.com/docs/3.3/

Let's start with the top of the page. In Kelvin's design, we use a circular image with centered title and description below:

image

The tag page template actually has 3 different layouts, depending on:

1. https://publiclab.org/tag/air-quality if there is a wiki page with the same name
2. https://publiclab.org/tag/coqui - if there is a lead image on that wiki page 
3. https://publiclab.org/tag/none if there is no matching wiki page at all

image

All three could follow the same new centered layout, where the circle could be grey if there's no image. The template for this page can be found here:

https://github.com/publiclab/plots2/blob/master/app/views/tag/show.html.erb

We'd love help with this!

### PR DESCRIPTION
**Pull request by oorjitchowdhary**
Fixes #4111 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by oorjitchowdhary** [comment]
@publiclab/reviewers I'm getting this error.. But my code seems to be correct..
screenshot 46

**Comment by oorjitchowdhary** [comment]
https://gist.github.com/oorjitchowdhary/264093dd0b989d7054574f9099e9e6db
Here is my gist

**Comment by oorjitchowdhary** [comment]
@publiclab/reviewers A little bit of help here plss..

File: app/views/tag/show/_header.html.erb [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+      			<div id="outer">
+          		    <div class="button_slide slide_left"><i class="fas fa-external-link-alt"></i> Follow Topic</a></div>
+      			</div>
+    		       <% end %>
+      		</div>
+
+	    </div>
   </div>
+
 <% elsif @wildcard %>
```
Comment (jywarren) [inline app/views/tag/show/_header.html.erb]: Aha - so you removed the conditional code above, so this `elsif` is left alone here! I think you may have to preserve the `if @wiki` conditionals to make this work!

**Comment by SidharthBansal** [comment]
Please provide the screenshot once the issue is resolved which you are currently facing

**Pull request closed without merging**